src/BoostedEventShapeTagger.cc

```diff
@@ -482,7 +496,7 @@ void BoostedEventShapeTagger::getJetValues( const pat::Jet& jet ){
     m_BESTvars["et"]      = thisJet.Pt();
     m_BESTvars["eta"]     = thisJet.Rapidity();
     m_BESTvars["mass"]    = thisJet.M();
-    m_BESTvars["SDmass"]  = jet.userFloat("ak8PFJetsPuppiSoftDropMass");
+    m_BESTvars["SDmass"]  = jet.mass();
```

Comment:
So here don't use the softdrop mass anymore? Which one did you from MiniAOD?

Reply:
Or do you use "ak8PFJetsPuppiValueMap:mass" from MiniAOD? Then I think it is the same as `jet.mass()` here.

Reply:
Sorry for the confusion, in the midst of checking the inputs I thought there was some difference in the mass, and was testing different userFloats.  I did another check just now and get consistent results with the soft drop mass (it was consistent before as I changed the inputs in both places to the raw PUPPI mass).  I reverted this change.

Reply:
@justinrpilot 

I think if you use `float puppiSDmass = (puppi_subjet0+puppi_subjet1).M();` for the training, then the most consistent one would be `jet.groomedMass()` [0] which are also computed from the subjets (`ak8PFJetsPuppiSoftDropMass` is computed from subjets before applying the JEC though).

[0] https://github.com/cms-sw/cmssw/blob/master/DataFormats/PatCandidates/interface/Jet.h#L527

Reply:
I realized that the puppi subjets stored in MiniAOD does not have the latest JEC applied as the subjets reclustered by JetToolBox, so if you have already seen good agreement with `ak8PFJetsPuppiSoftDropMass` then maybe it's better to just stick to that. I expect the effect on the final DNN output is anyhow very small, right?

Reply:
Yes, I saw good agreement with ````ak8PFJetsPuppiSoftDropMass```` -- it should have a small effect and I agree we can go ahead with this.